crates/pathfinder/src/storage/schema/revision_0010.rs

```diff
@@ -298,6 +293,254 @@ mod tests {
 
                     Ok(())
                 }
+
+                /// Deletes all rows from __head down-to reorg_tail__
+                /// i.e. it deletes all rows where `block number >= reorg_tail`.
+                pub fn reorg(
+                    connection: &Connection,
+                    reorg_tail: StarknetBlockNumber,
+                ) -> anyhow::Result<()> {
+                    connection.execute(
+                        "DELETE FROM starknet_blocks WHERE number >= ?",
+                        rusqlite::params![reorg_tail.0],
+                    )?;
+                    Ok(())
+                }
+
+                /// Returns the [number](StarknetBlockNumber) of the latest block.
+                pub fn get_latest_number(
+                    connection: &Connection,
+                ) -> anyhow::Result<Option<StarknetBlockNumber>> {
+                    use anyhow::Context;
+
+                    let mut statement = connection.prepare(
+                        "SELECT number FROM starknet_blocks ORDER BY number DESC LIMIT 1",
```

Comment:
I just used `max(number)` in the migration which I think was the best option when we last looked at this?